graph/api/src/main/java/org/jboss/windup/graph/service/FileService.java

```diff
@@ -24,6 +25,11 @@ public FileModel createByFilePath(String filePath)
         return createByFilePath(null, filePath);
     }
 
+    public ApplicationInputPathModel createInputPath(String filePath)
```

Comment:
This is really the wrong place for this method. Maybe it should be on a subclass of this service?

Reply:
Right. I think will do few changes as per our earlier considerations, including this one.